sopel_waifu/waifu.json

```
@@ -1238,34 +1238,87 @@
     ],
     "Super Sonico The Animation": [
         "Super Sonico",
```

Per other discussion, this suggestion implements the strategy:

1. Rename franchise to just "Super Sonico"
2. Keep Pochaco
3. At a later date, @dgw will fill in more characters

```suggestion
    "Super Sonico": [
        "Super Sonico",
        "Super Pochaco",
```